### ISSUE TITLE
Python class member functions are not qualified with their corresponding class name

### ISSUE THREAD

**Issue by tobias-klein** [terryyin/lizard#125]
Python functions that are members of a class are not qualified with their corresponding class name. The qualification is done in case of C++ functions and should also be done for Python functions.

**Comment by terryyin** [comment]
Yeah, I noticed this too. Will fix this with priority (but still i'm quite busy with other project(.

**Comment by rakhimov** [comment]
Should there be Pythonic separator dot '.' or keep it C++ style double colon?

**Comment by terryyin** [comment]
Good question. I will see if I can make the connectors right.

**Comment by tobias-klein** [comment]
@rakhimov: I think it would make most sense to display it in the language-specific style. So I would vote for the "pythonic" separator dot '.'
